flytekit/extras/tensorflow/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+Flytekit Tensorflow
+=========================================
+.. currentmodule:: flytekit.extras.tensorflow
+
+.. autosummary::
+   :template: custom.rst
+   :toctree: generated/
+"""
+from flytekit.loggers import logger
+
+# TODO: abstract this out so that there's an established pattern for registering plugins
+# that have soft dependencies
+try:
+    # isolate the exception to the torch import
+    import tensorflow
+
+    _tensorflow_installed = True
+except (ImportError, OSError):
+    _tensorflow_installed = False
+
+
+if _tensorflow_installed:
+    from .model import TensorflowModelTransformer
```

Comment:
```suggestion
    from .module import TensorFlowModuleTransformer
```

Can you change the file name to `module.py` and the transformer name to `TensorFlowModuleTransformer`?